src/app/modules/activities-management/pages/activities-management.component.scss

```diff
@@ -0,0 +1,9 @@
+.activity-container{
+    width: 100%;
```

Comment:
bootstrap has a w-100 class which does this

Reply:
Yes bootstrap has those classes, but I created this class to use the media-query. On desktop screen we need 80% of width, but in phone we need 100%. Kind of style conditionally.

Reply:
I am not sure if can do it, only with bootstrap classes, without using empty divs.

Reply:
You gave me the idea. I could do it with bootstrap classes only. Thanks